clang/include/clang/Basic/OpenMPKinds.h

```diff
@@ -257,6 +264,10 @@ struct OMPInteropInfo final {
   llvm::SmallVector<Expr *, 4> PreferTypes;
 };
 
+unsigned getOpenMPDefaultVariableCategory(StringRef Str,
+                                          const LangOptions &LangOpts);
+const char *getOpenMPDefaultVariableCategoryName(unsigned VC);
```

Comment:
use the actual type here instead of an `unsigned` such that the cast can be avoided.